modules/analytics/src/views/full/TimeSeriesChart.tsx

```diff
@@ -3,14 +3,16 @@ import cx from 'classnames'
 import _ from 'lodash'
 import moment from 'moment'
 import React, { FC } from 'react'
-import { Area, AreaChart, ResponsiveContainer, Tooltip, XAxis } from 'recharts'
+import { Area, AreaChart, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts'
 
 import { MetricEntry } from '../../backend/typings'
 
-import { Extras } from './index'
+import { Channel, Extras } from './index'
 import style from './style.scss'
+import { mergeDataForCharts } from './utils'
 
 const CHANNEL_COLORS = {
+  api: '#4A154B',
```

Comment:
don't we support ms teams ?

Reply:
Yup but defining a color is not necessary, it will still be displayed